wordclock/http_server.c

```diff
@@ -845,8 +845,6 @@ static void handle_hw_cfg_post(int s, wificfg_method method,
                     sdk_system_restart();
             	} else if (strcmp(buf, "FactoryReset") == 0){
             	    wificfg_write_string(s, http_redirect_header_delayed);
-                    struct sdk_station_config config = {"", "", 0, {0}};
```

Comment:
Een FactoryReset zou ook de wifi settings moeten weg gooien,
In normaal gebruik zal je ook niet snel dit doen maar met debugging is het erg handig dat de klok echt alles vergeten is

Reply:
ik vond het just onhandig. Wifi settings zijn los van de andere settings. Als je van de wifi connection af wil, kun je die in de wifi settings handmatig vernaggelen, komt op hetzelfde neer

Reply:
FactoryReset is nu niet echt een FactoryReset.
maar prima, ik zal m'n settings wel vernaggelen